docs/config.rst

```diff
@@ -110,7 +110,20 @@ Core configuration
     The original MPD server only supports 10000 tracks in the tracklist. Some
     MPD clients will crash if this limit is exceeded.
 
-
+.. confval:: core/restore_state
```

Comment:
Changing the name is both: hard and easy. Easy to do, and hard to find a good one. If you have a good/better name, we can use it. 
By the way, if you write 'we', do you mean 'you and me' or 'you and other mopidy maintainer'?